app/assets/v2/js/pages/bounty_details.js

```diff
@@ -1051,27 +1067,44 @@ const build_uri_for_pull_bounty_from_api = function() {
   return uri;
 };
 
+const get_bounty_by_id = function(bountyId) {
```

Comment:
this seems out of scope for this PR... were you running into some issue with looking up by URL (even if it's a non github URL)?